rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -210,15 +210,21 @@ def most_likely_entity(self, idx, entities):
         else:
             return "", 0.0
 
-    def _create_entity_dict(self, tokens, start, end, entity, confidence):
+    def _create_entity_dict(self, message, tokens, start, end, entity, confidence):
```

Comment:
Could you add types here and a docstring?

Reply:
fixed here https://github.com/RasaHQ/rasa/pull/4018/commits/f65f354a4d115ce52f075aed3249740b81b96601